CRM/Speakcivi/Page/Speakcivi.php

```diff
@@ -152,19 +152,22 @@ public function petition($param) {
       $this->setContactCreatedDate($contact['id'], $param->create_dt);
     }
 
+    $opt_in_for_activity_status = $this->opt_in;
```

Comment:
Temp variable is used because opt_in can't be changed to value `0`. Only on `1` confirmation email is sending.

You're absolutely right. I will use camelCase convention also for variables.